backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -194,7 +194,9 @@ public IAsyncEnumerable<PartOfSpeech> GetPartsOfSpeech()
             .Select(partOfSpeech => new PartOfSpeech
             {
                 Id = partOfSpeech.Guid,
-                Name = FromLcmMultiString(partOfSpeech.Name)
+                Name = FromLcmMultiString(partOfSpeech.Name),
+                // TODO: Abreviation = FromLcmMultiString(partOfSpeech.Abreviation),
+                Predefined = true, // TODO: Determine by GUID, but for now we can assume all FW parts of speech are predetermined
```

Comment:
In order to be able to determine if a part of speech coming from .fwdata is predetermined or not, we will need at least a list (or hashset) of the GUIDs from the GOLDEtic.xml file. For now, though, we can assume that custom parts of speech not listed in GOLDEtic.xml are extremely rare, and so anything coming from .fwdata can be assumed to be predetermined.

Reply:
The `!string.IsNullOrEmpty(partOfSpeech.CatalogSourceId)` check we use elsewhere doesn't work here: if the part of speech came from CRDT originally and ended up in FW due to a sync, it might have had `Predetermined = true` in CRDT but would not have a catalog source ID in LCM. This is an area we might need to improve on in future work.

Reply:
I think that makes sense to me, although we need to be consistent. Right now `GetPartsOfSpeech` and `UpdatePartOfSpeech` use different code to transform parts of speech into miniLcm objects, I don't want any duplicate code there they should be using the same helper function.

Reply:
Made them consistent in c1bcffa9. Currently just set Predefined to `true`; later on we'll load the GUIDs from GOLDEtic.xml and make a lookup table out of them or something.